File: Actividad 2/actividad213.py
valcomp = float(input("Ingrese el valor de la compra: "))
color = input("Color de la bolita: ")

if color == "BLANCO":
    pdes = 0
    valpag = valcomp

elif color == "VERDE":
    pdes = 10
    valpag = valcomp - ((valcomp * 10 )/100)
elif color == "AMARILLO":
    pdes = 25
    valpag = valcomp - ((valcomp * pdes )/100)
elif color == "AZUL":
    pdes = 50
    valpag = valcomp - ((valcomp * pdes )/100)
elif color == 'ROJO':
    pdes = 100
    valpag = valcomp - ((valcomp * pdes )/100)

print(valpag)

File: Actividad 2/actividad213.java
import java.util.Scanner;

public class actividad213 {
    public static void main(String[] args) {
        Scanner scanner = new Scanner(System.in);
        System.out.print("Ingrese el valor de la compra: ");
        float VALCOMP = Float.parseFloat(scanner.nextLine());
        System.out.print("Color de la bolita: ");
        String COLOR = scanner.nextLine().toUpperCase();

        float VALPAG;
        int PDES;

        switch (COLOR) {
            case "BLANCO":
                PDES = 0;
                VALPAG = VALCOMP;
                break;
            case "VERDE":
                PDES = 10;
                VALPAG = VALCOMP - ((VALCOMP * 10) / 100);
                break;
            case "AMARILLO":
                PDES = 25;
                VALPAG = VALCOMP - ((VALCOMP * PDES) / 100);
                break;
            case "AZUL":
                PDES = 50;
                VALPAG = VALCOMP - ((VALCOMP * PDES) / 100);
                break;
            case "ROJO":
                PDES = 100;
                VALPAG = VALCOMP - ((VALCOMP * PDES) / 100);
                break;
            default:
                System.out.println("Color no válido");
                return;
        }

        System.out.println(VALPAG);
    }
}
